src/auth/screens/auth-profile.screen.js

```diff
@@ -113,6 +121,34 @@ class AuthProfile extends Component {
     }
   };
 
+  signOut() {
+    const { signOutByDispatch, navigation } = this.props;
+
+    const resetAction = NavigationActions.reset({
+      index: 0,
+      key: null,
+      actions: [NavigationActions.navigate({ routeName: 'Login' })],
+    });
+
+    signOutByDispatch().then(() => {
+      const url = 'https://github.com/logout';
+
+      navigation.dispatch(resetAction);
+      this.openLink(url);
+    });
+  }
+
+  openLink = url => {
+    if (Platform.OS === 'ios') {
+      SafariView.show({
+        url,
+        fromBottom: true,
+      });
+    } else {
+      Linking.openURL(url);
+    }
+  };
```

Comment:
I feel like openLink is a fairly complex method that will (and should) likely be used in other places. Not sure where it should be moved to, though, maybe `utils`?